packages/experiments/src/components/CollapsibleSection/CollapsibleSectionTitle.ts

```diff
@@ -1,9 +1,9 @@
-import { createComponent, ISlottableComponentType } from '../../Foundation';
+import { createComponent } from '../../Foundation';
 import { CollapsibleSectionTitleView as view } from './CollapsibleSectionTitle.view';
 import { getStyles as styles } from './CollapsibleSectionTitle.styles';
 import { ICollapsibleSectionTitleProps } from './CollapsibleSectionTitle.types';
 
-export const CollapsibleSectionTitle: ISlottableComponentType<ICollapsibleSectionTitleProps> = createComponent({
+export const CollapsibleSectionTitle: React.StatelessComponent<ICollapsibleSectionTitleProps> = createComponent({
```

Comment:
Hmm, this generated a deprecated warning in other areas. @jdhuntington Do you know of those rules are applying to all packages?